### DISCUSSION TITLE
Any update

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#52] [General]
Dhalinyaro xagee ayuu wax idin maraa ? waqtiga nagu socda.

Qolada server-ka ma bilaabeen howshooda ?

Qolada extension-kana xagee marinaayaan ?

@dugsiiyeinc/dhaqsopay-extension-developers

@dugsiiyeinc/dhaqsopay-server-developers

@dugsiiyeinc/dhaqsopay-website-developers

**Comment by AishaKeyf** [comment]
@sharafdin  webkii waa dhammeenay,  kaliyaa wxaa ku kal ra' yi duwanaanay qaabka colors ka loogu fadhiisinayo oo aan isku raacnay in zoomka ugu horreeya aan kaga wada tashanno iyo api documentation

**Reply by sharafdin** [reply]
hada pull request waan aqbalaa right ?

**Reply by AishaKeyf** [reply]
@sharafdin May be, it's OK

**Comment by faaraxcabdulaahi** [comment]
> Dhalinyaro xagee ayuu wax idin maraa ? waqtiga nagu socda.
> 
> Qolada server-ka ma bilaabeen howshooda ?
> 
> Qolada extension-kana xagee marinaayaan ?
> 
> @dugsiiyeinc/dhaqsopay-extension-developers
> 
> @dugsiiyeinc/dhaqsopay-server-developers
> 
> @dugsiiyeinc/dhaqsopay-website-developers

Eng qaybtaa  ka shaqeynayay extensionka ee top-Upka gabogabo buu ii marayaa.
